### PR TITLE
Fix building on windows issue w/ cross-env

### LINKED ISSUE #274: Building storybook does not work on windows
As reported by @stephyswe on Discord: 

>>>
1.  Prepare your design system for export - BABEL_ENV needs "set" to work for Windows Users, also --copy-files to add .mdx files in /dist/ folder
REPLACE
{
  "scripts": {
    "build": "BABEL_ENV=production babel src -d dist",
WITH
{
  "scripts": {
    "build": "set BABEL_ENV=production&& babel src -d dist --copy-files",
...
>>>

### PR DESCRIPTION
**Pull request by tmeasday**
Fixes #274

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@tmeasday i'm going to take this time to go through this and also check a issue that @DaleSeo mentioned also regarding the src folder usage with svn. 

I'm going to start this tomorrow(tuesday), with a "fresh pair of eyes".

File: content/design-systems-for-developers/react/en/distribute.md [hunk 1]
```diff
@@ -58,18 +58,18 @@ export * from './Icon';
 export * from './Link';
 ```
 
-Let’s add a development dependency on `@babel/cli` to compile our JavaScript for release:
+Let’s add a development dependency on `@babel/cli` and `cross-env` to compile our JavaScript for release:
 
 ```bash
-yarn add --dev @babel/cli
+yarn add --dev @babel/cli cross-env`
```
Comment (jonniebigodes) [inline content/design-systems-for-developers/react/en/distribute.md]: @tmeasday if you don't mind, drop the extra tick after cross-env

**Comment by tmeasday** [comment]
Hmm, I think someone mentioned seeing this before. We should figure it out I guess.

File: content/design-systems-for-developers/react/en/distribute.md (hunk 1 above)
Comment (tmeasday) [reply content/design-systems-for-developers/react/en/distribute.md]:
```suggestion
yarn add --dev @babel/cli cross-env
```

**Comment by jonniebigodes** [comment]
@tmeasday merge when you're able.

Regarding my problem with the changelog, i created a issue on the auto repo and they fixed it. It's working now. I was able to continue from that step onwards to the end without a hitch. 

There's some tweaks to the tutorial that are indirectly related to this pr/issue that need to be addressed. But for this one we're good to go.

**Comment by tmeasday** [comment]
Thanks @jonniebigodes

**Pull request merged**